react/Interface.tsx

```diff
@@ -1,143 +1,32 @@
 import React from 'react'
-import { FormattedMessage, useIntl } from 'react-intl'
 import {
   createSystem,
-  DataGrid,
-  DataView,
-  DataViewControls,
-  FlexSpacer,
   PageHeader,
   PageTitle,
-  Pagination,
-  Search,
-  Tag,
-  useDataGridState,
-  useDataViewState,
-  usePaginationState,
-  useSearchState,
+  
 } from '@vtex/admin-ui'
-import faker from 'faker'
-import { columns, messages, table } from './utils/definedMessages'
-import { CURRENCY, ITEMS_PER_PAGE, LOCALE, NUMBER_OF_ITEMS } from './utils/constants'
+import ProviderInterface from './provider/providerInterface'
+import Table from './components/Table'
+import { FormattedMessage } from 'react-intl'
+import { messages } from './utils/definedMessages'
 
 const [ThemeProvider] = createSystem({
   key: 'admin-ui-example',
```

Comment:
Is this changed in future PR's?

Reply:
It is a good idea to change this to something unique to the application! [The admin-ui docs](https://admin-ui.vercel.app/teams/developing/getting-started) specifies that it needs to be a unique key.

```
// 2. add an unique key for the onda instance
const [ThemeProvider] = createSystem({
  key: 'unique-key-in-kebab-case',
})
```